vm/src/vm.rs

```diff
@@ -1045,6 +1046,19 @@ impl VirtualMachine {
                 }
                 None
             }
+            bytecode::Instruction::UnpackSequence { size } => {
+                let value = self.pop_value();
+
```

Comment:
Should we add a type check here?

Reply:
The full implementation requires ```__iter__```, which you've just started to provide. I'll extend this to call that.